common-moonmage.lic

```diff
@@ -143,12 +143,13 @@ module DRCMM
   # Returns true if dropped something, false otherwise.
   def drop_moon_weapon?
     moon_weapon_regex = /((black|red-hot|blue-white) moon[\w]+)/
+    moon_drop_messages = ["As you open your hand", "What were you referring to"]
     dropped_it = false
     if DRC.left_hand =~ moon_weapon_regex
-      dropped_it = dropped_it || DRC.bput("drop #{DRC.left_hand}", "As you open your hand", "What were you referring to") == "As you open your hand"
+      dropped_it = dropped_it || DRC.bput("drop #{DRC.left_hand}", *moon_drop_messages) == "As you open your hand"
     end
     if DRC.right_hand =~ moon_weapon_regex
-      dropped_it = dropped_it || DRC.bput("drop #{DRC.right_hand}", "As you open your hand", "What were you referring to") == "As you open your hand"
+      dropped_it = dropped_it || DRC.bput("drop #{DRC.right_hand}", *moon_drop_messages) == "As you open your hand"
```

Comment:
Consistency change: moved the messages to match on to an array to mirror how the `wear_moon_weapon?` is structured.